.coafile

```diff
@@ -22,3 +22,8 @@ bears = MarkdownBear, AlexBear
 
 [Git]
 bears = GitCommitBear
+
+[YAML]
+files = **/*.yml
```

Comment:
I think **.yml would do the work too.

Reply:
I'm not sure if this is due to a bug in our globbing module... maybe it was already fixed^^ But please check if coala grabs all desired files (you can do that by executing coala in debug mode, it will list all files analyzed)